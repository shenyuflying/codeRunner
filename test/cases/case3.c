/* test for commands */
help
int a;
list
puts("hello");
print
clear
print
list

h
int a;
l
puts("hello");
p
c
p
l
exit
