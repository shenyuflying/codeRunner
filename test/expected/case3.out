"list" "l" list the c code buffer
"clear" "c" clear the c code buffer
"print" "p" print the old output buffer
"quit" "q" quit the codeRunner
"exit" "e" quit the codeRunner
"help" "h" show the help
/* test for commands */
int a;
hello
hello
"list" "l" list the c code buffer
"clear" "c" clear the c code buffer
"print" "p" print the old output buffer
"quit" "q" quit the codeRunner
"exit" "e" quit the codeRunner
"help" "h" show the help
int a;
hello
hello
bye!
